### PR TITLE
modify the method to get env var for db

### PR DESCRIPTION
**Pull request by emya**
[#149453239]

### CODE REVIEW

File: lib/etl/config.rb [hunk 1]
```diff
@@ -29,17 +29,17 @@ def db(&b)
       @db
     end
 
-    def database_env_vars
+    def database_env_vars(prefix="ETL_DATABASE")
       conn_params = {}
       conn_params[:encoding] = "utf8"
```
Comment (ChrisPRobinson) [inline lib/etl/config.rb]: Its possible you might not want the conn_params that I'm setting that isn't pull from envs. They are postgres specific I think.

Comment (emya) [inline lib/etl/config.rb]: This is not specific to Postgres, so we should use this.

**Comment by ChrisPRobinson** [comment]
Also please refactor the redshift piece to extract its stuff that builds the parameters from env_vars too

File: lib/etl/config.rb (hunk 1 above)
Comment (ChrisPRobinson) [inline lib/etl/config.rb]: ok, fine with me then

**Pull request merged**